src/OrbitQt/CallTreeWidget.ui

```diff
@@ -38,20 +57,33 @@
      </item>
     </layout>
    </item>
-   <item row="1" column="0">
-    <widget class="CustomSignalsTreeView" name="callTreeTreeView">
-     <property name="contextMenuPolicy">
-      <enum>Qt::CustomContextMenu</enum>
-     </property>
-     <property name="alternatingRowColors">
-      <bool>true</bool>
-     </property>
-     <property name="selectionMode">
-      <enum>QAbstractItemView::ExtendedSelection</enum>
-     </property>
-     <property name="sortingEnabled">
-      <bool>true</bool>
+   <item row="0" column="0">
+    <widget class="QWidget" name="noticeWidget" native="true">
```

Comment:
I would call this `inspectionNoticeWidget` as `noticeWidget` alone sounds too generic to me.

Reply:
Done.